lib/find-view.js

```diff
@@ -0,0 +1,737 @@
+const _ = require('underscore-plus');
+const {TextBuffer, TextEditor, CompositeDisposable} = require('atom');
+const Util = require('./project/util');
+const etch = require('etch');
+
+module.exports =
+class FindView {
+  constructor(model, {findBuffer, replaceBuffer, findHistoryCycler, replaceHistoryCycler} = {}) {
+    this.model = model;
+    this.findBuffer = findBuffer;
+    this.replaceBuffer = replaceBuffer;
+    this.findHistoryCycler = findHistoryCycler;
+    this.replaceHistoryCycler = replaceHistoryCycler;
+    this.subscriptions = new CompositeDisposable();
+
+    etch.initialize(this)
+
+    this.findHistoryCycler.addEditorElement(this.findEditor.getElement());
+    this.replaceHistoryCycler.addEditorElement(this.replaceEditor.getElement());
+    this.handleEvents();
+    this.clearMessage();
+    this.updateOptionViews();
+    this.updateReplaceEnablement();
+    this.createWrapIcon();
+  }
+
+  update(props) {}
+
+  render() {
+    return (
+      etch.dom.div({tabIndex: -1, className: 'find-and-replace'},
+        etch.dom.header({className: 'header'},
+          etch.dom.span({ref: 'descriptionLabel', className: 'header-item description'},
```

Comment:
Currently breaking here:

~~~
Uncaught TypeError: etch.dom.span is not a function @ find-view.js:33
	render @ find-view.js:33
	initialize @ component-helpers.js:60
	FindView @ find-view.js:16
	createViews @ find.coffee [sm]:141
	(anonymous function) @ find.coffee [sm]:42
	module.exports.CommandRegistry.handleCommandEvent @ command-registry.coffee:255
	(anonymous function) @ command-registry.coffee:1
	module.exports.KeymapManager.dispatchCommandEvent @ keymap-manager.js:599
	module.exports.KeymapManager.handleKeyboardEvent @ keymap-manager.js:390
	module.exports.WindowEventHandler.handleDocumentKeyEvent @ window-event-handler.coffee:76
	(anonymous function) @ window-event-handler.coffee:1
~~~